src/config.ts

```diff
@@ -0,0 +1,171 @@
+    * - Chrome: 1.5
+    * - Edge: 1.75
+    * - Firefox: 0.9
+    * - Safari: 0.95
+    *
+    * @since 2.0.0
+    * @type Number
+    * @default 1
+    */
+    browserShadowBlurConstant = 1
+
+
+    /**
+     * Pixel per Inch as a default value set to 96. Can be changed for more realistic conversion.
+     */
+    DPI = 96
+
+    /**
+     * Device Pixel Ratio
+     * @see https://developer.apple.com/library/safari/documentation/AudioVideo/Conceptual/HTML-canvas-guide/SettingUptheCanvas/SettingUptheCanvas.html
+     */
+    devicePixelRatio = 1
+
+    /**
+     * Pixel limit for cache canvases. 1Mpx , 4Mpx should be fine.
+     * @since 1.7.14
+     * @type Number
+     * @default
+     */
+    perfLimitSizeTotal = 2097152
+
+    /**
+     * Pixel limit for cache canvases width or height. IE fixes the maximum at 5000
+     * @since 1.7.14
+     * @type Number
+     * @default
+     */
+    maxCacheSideLimit = 4096
+
+    /**
+     * Lowest pixel limit for cache canvases, set at 256PX
+     * @since 1.7.14
+     * @type Number
+     * @default
+     */
+    minCacheSideLimit = 256
+
+    /**
+     * When 'true', style information is not retained when copy/pasting text, making
+     * pasted text use destination style.
+     * Defaults to 'false'.
+     * @type Boolean
+     * @default
+     * @deprecated
+     */
+    disableStyleCopyPaste = false
+
+    /**
+     * Enable webgl for filtering picture is available
+     * A filtering backend will be initialized, this will both take memory and
+     * time since a default 2048x2048 canvas will be created for the gl context
+     * @since 2.0.0
+     * @type Boolean
+     * @default
+     */
+    enableGLFiltering = true
+
+    /**
+     * if webgl is enabled and available, textureSize will determine the size
+     * of the canvas backend
+     * 
+     * In order to support old hardware set to `2048` to avoid OOM
+     * 
+     * @since 2.0.0
+     * @type Number
+     * @default
+     */
+    textureSize = 4096
+
+    /**
+     * Skip performance testing of setupGLContext and force the use of putImageData that seems to be the one that works best on
+     * Chrome + old hardware. if your users are experiencing empty images after filtering you may try to force this to true
+     * this has to be set before instantiating the filtering backend ( before filtering the first image )
+     * @type Boolean
+     * @default false
+     */
+    forceGLPutImageData = false
+
+    /**
+     * WebGL
+     */
+    maxTextureSize?: number
+
+    /**
+     * WebGL
+     */
+    webGLPrecision?: TWebGLPrecision
+
+    /**
+     * If disabled boundsOfCurveCache is not used. For apps that make heavy usage of pencil drawing probably disabling it is better
+     * @default true
+     */
+    cachesBoundsOfCurve = true
+
+    /**
+     * Map of font files
+     * Map<fontFamily, pathToFile> of font files
+     */
+    fontPaths: Record</** fontFamily */ string, /** pathToFile */ string> = {}
```

Comment:
I don't really understand for this is used for
Can it be made a local var in `canvas#toSVG`?
It's the only place it is used....
I thought exposing `registerFonts` that will add fonts to `fontPaths` and call register fonts on node canvas.
??

Reply:
this can be made local to the svg export function yes. But for this PR we should leave it here i guess.

Reply:
only used by canvas.toSVG